src/sphinx_contributors/__init__.py

```diff
@@ -72,14 +72,17 @@ class ContributorsDirective(Directive):
     def run(self):
         limit = self.options.get("limit", 10)
         order = self.options.get("order", "DESC") == "DESC"
+        providers = self.options.get("providers", "github")
         exclude = self.options.get("exclude", "").split(",")
         use_avatars = "avatars" in self.options
         show_contributions = "contributions" in self.options
 
         contributors = []
+        api_url = "api.github.com" if providers == "github" else "gitlab.api.com"
```

Comment:
``gitlab.api.com`` -> The URL does not seem to be valid. Besides, the methods and responses between the GitHub and GitLab API do not seem to be compatible.
For more information, you can check https://docs.gitlab.com/ee/api/